tests/test_fp8/test_fp8_all_to_all_single.py

```diff
@@ -0,0 +1,34 @@
+import torch
+import torch.distributed as dist
+from torch.distributed.distributed_c10d import _get_default_group
+from torch.testing import assert_close
+
+from colossalai import launch
+from colossalai.accelerator import get_accelerator
+from colossalai.shardformer.layer._operation import _all_to_all_single
+from colossalai.testing import parameterize, rerun_if_address_is_in_use, spawn
+
+
+@parameterize("shape", [(1, 8, 16)])
```

Comment:
In this test, the first dimension of the input shape is preferably greater than 1.

Reply:
Fixed